src/Umbraco.Web/PublishedContentExtensions.cs

```diff
@@ -97,8 +102,10 @@ public static string GetTemplateAlias(this IPublishedContent content)
         /// <remarks>Returns true if <c>GetProperty(alias, recurse)</c> is not <c>null</c> and <c>GetProperty(alias, recurse).HasValue</c> is <c>true</c>.</remarks>
         public static bool HasValue(this IPublishedContent content, string alias, bool recurse)
         {
-            var prop = content.GetProperty(alias, recurse);
-            return prop != null && prop.HasValue();
+            throw new NotImplementedException("WorkInProgress");

```

Comment:
There's a few of these in this PR, if these are going to remain then this known issues list needs to be updated http://issues.umbraco.org/issue/U4-11279